lib/src/screens/memory/memory_service.dart

```diff
@@ -178,4 +178,27 @@ class MemoryService {
     );
     updateState1(memory);
   }
+
+  Future<void> disconnect() async {
```

Comment:
this is moved to appmenu service I think. @amit-jha-cmd correct ?

Reply:
yes.